js/build/build-recipes.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs')
+
+const sass = require('node-sass')
+
+const input = process.argv[2]
+const output = process.argv[3]
+
+if (!input || !output) {
+  throw new Error('Input/Output arguments not given.')
+}
+
+const scssResult = sass.renderSync({
+  file: input,
+  sourceMap: true,
+  outputStyle: 'nested',
+  outFile: output
+})
+
+fs.writeFileSync(`${output}`, scssResult.css)
```

Comment:
Since this is just a variable, can the interpolation step be avoided?
```suggestion
fs.writeFileSync(output, scssResult.css)
```